pokecli.py

```diff
@@ -249,6 +249,16 @@ def init_config():
     add_config(
         parser,
         load,
+        short_flag="-r",
```

Comment:
This should be a task level configuration option. Check out IncubateEggs for an example